ContactManager/ContactManager/main.swift

```diff
@@ -0,0 +1,84 @@
+//
+//  main.swift
+//  CantactManager
+//
+//  Created by 박재우 on 2022/12/21.
+//
+
+import Foundation
+
+func split(input: String) throws -> (String, String, String) {
+    var splitted = input.components(separatedBy: " / ")
+    if splitted.count != 3 {
+        splitted = input.components(separatedBy: "/")
+        if splitted.count != 3 {
+            throw ContactManagerError.invalidInput
+        }
+    }
+    return (splitted[0], splitted[1], splitted[2])
```

Comment:
그럼 여기서 0,1,2 가 무슨 의미인지 모를듯 합니다 :)
저라면 아래처럼 변수로 선언하고 사용할 것 같아요.

```suggestion
    let indexOfName = 0
    let indexOfAge = 1
    let indexOfPhoneNumber = 2
    return (splitted[indexOfName], splitted[indexOfAge], splitted[indexOfPhoneNumber])
```